sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeTestUtils.scala

```diff
@@ -26,6 +26,16 @@ object DateTimeTestUtils {
 
   val ALL_TIMEZONES: Seq[TimeZone] = TimeZone.getAvailableIDs.toSeq.map(TimeZone.getTimeZone)
 
+  val TIMEZONES = Seq(
```

Comment:
shall we move it to `SparkFunSuite` and name it like `outstandingTimezones`?

Reply:
It seems it should be useful across a few test suites. I will do that.